### PR TITLE
release-notes: add `--skip-first-commit`/`-s` option

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
This option can be used together with `--start-rev` to skip the first commit. This is helpful if the `--start-rev` is a tag which should not be included in the release notes generation.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Added `--skip-first-commit`/`-s` option to the `release-notes` tool.
```

### CODE REVIEW

File: pkg/notes/options/options.go [hunk 1]
```diff
@@ -228,6 +237,15 @@ func (o *Options) ValidateAndFinish() (err error) {
 			if err != nil {
 				return fmt.Errorf("resolving %s: %w", o.StartRev, err)
 			}
+
+			if o.SkipFirstCommit {
+				logrus.Infof("Skipping first commit: %s", sha)
+				sha, err = repo.NextCommit(sha, git.DefaultRemote+"/"+o.Branch)
+				if err != nil {
+					return fmt.Errorf("get next repo commit: %w", err)
```
Comment (xmudrii) [inline pkg/notes/options/options.go:245]: Just a tiny nit:
```suggestion
					return fmt.Errorf("getting the next repo commit: %w", err)
```

**Review by xmudrii — CHANGES_REQUESTED** [review]

File: pkg/notes/options/options.go (hunk 1 above)
Comment (saschagrunert) [reply pkg/notes/options/options.go]: Fixed as suggested.

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Comment by saschagrunert** [comment]
Rebased to get the CI changes in.

**Comment by xmudrii** [comment]
Looks like Prow doesn't see the LGTM label 👀 
/lgtm cancel

**Pull request merged**